src/OrchardCore.Modules/OrchardCore.ContentFields/Views/LocalizationSetContentPickerField.Edit.cshtml

```diff
@@ -0,0 +1,72 @@
+@model OrchardCore.ContentFields.ViewModels.EditLocalizationSetContentPickerFieldViewModel
+@using OrchardCore.ContentManagement.Metadata.Models
+@using OrchardCore.ContentFields.Settings;
+@using Newtonsoft.Json;
+@using Newtonsoft.Json.Serialization;
+@using OrchardCore.Environment.Shell;
+
+@inject ShellSettings ShellSettings
+
+@{
+    var settings = Model.PartFieldDefinition.Settings.ToObject<LocalizationSetContentPickerFieldSettings>();
+    var selectedItems = JsonConvert.SerializeObject(Model.SelectedItems, new JsonSerializerSettings { ContractResolver = new CamelCasePropertyNamesContractResolver() });
+    var partName = Model.PartFieldDefinition.PartDefinition.Name;
+    var fieldName = Model.PartFieldDefinition.Name;
+    var tenantPath = string.IsNullOrEmpty(ShellSettings.RequestUrlPrefix) ? string.Empty : "/" + ShellSettings.RequestUrlPrefix;
+    var vueElementId = $"LocalizedContentPicker_{partName}_{fieldName}_{Guid.NewGuid().ToString("n")}";
+    var multiple = settings.Multiple.ToString().ToLowerInvariant();
+}
+
+<script asp-src="https://vuejs.org/js/vue.min.js" debug-src="https://vuejs.org/js/vue.js" asp-name="vuejs" at="Foot"></script>
+<script asp-src="https://unpkg.com/vue-multiselect@2.1.0/dist/vue-multiselect.min.js" asp-name="vuemultiselect" at="Foot"></script>
+<style asp-src="https://unpkg.com/vue-multiselect@2.1.0/dist/vue-multiselect.min.css"></style>
+<script asp-src="https://cdn.jsdelivr.net/sortable/1.4.2/Sortable.min.js" debug-src="https://cdn.jsdelivr.net/sortable/1.4.2/Sortable.js" asp-name="sortable" at="Foot"></script>
+<script asp-src="https://cdnjs.cloudflare.com/ajax/libs/Vue.Draggable/2.14.1/vuedraggable.min.js" debug-src="https://cdnjs.cloudflare.com/ajax/libs/Vue.Draggable/2.14.1/vuedraggable.js" asp-name="vuedraggable" depends-on="vuejs, sortable" at="Foot"></script>
+<script asp-src="~/OrchardCore.ContentFields/Scripts/vue-multiselect-wrapper.min.js" debug-src="~/OrchardCore.ContentFields/Scripts/vue-multiselect-wrapper.js" asp-name="vue-multiselect-wrapper" at="Foot" depends-on="vuejs, vuemultiselect, sortable"></script>
+<style asp-src="~/OrchardCore.ContentFields/Styles/vue-multiselect-wrapper.min.css" debug-src="~/OrchardCore.ContentFields/Styles/vue-multiselect-wrapper.css"></style>
+
+<label asp-for="LocalizationSets">@Model.PartFieldDefinition.DisplayName()</label>
+
+<div id="@vueElementId" class="vue-multiselect"  data-editor-type="LocalizationSetContentPicker" data-selected-items="@selectedItems" data-tenant-path="@tenantPath" data-search-url="ContentFields/SearchLocalizationSets?part=@partName&field=@fieldName" data-multiple="@multiple">
```

Comment:
I personally like this pattern to add a class and data-attributes to elements and use the  document.querySelectorAll() to find elements and instanciate the plugin. Is this a good pattern ? It removes the JS call at the bottom of the file and it's easier to wrap it in a Self-Executing Anonymous Function.